lib/appsignal/probes/erlang_probe.ex

```diff
@@ -0,0 +1,37 @@
+defmodule Appsignal.Probes.ErlangProbe do
+  def call do
+    io_metrics()
+    scheduler_metrics()
+    process_metrics()
+    memory_metrics()
+  end
+
+  defp io_metrics do
+    {{:input, input}, {:output, output}} = :erlang.statistics(:io)
+    Appsignal.set_gauge("erlang_io", Kernel.div(input, 1024), %{kind: "input"})
+    Appsignal.set_gauge("erlang_io", Kernel.div(output, 1024), %{kind: "output"})
+  end
+
+  defp scheduler_metrics do
+    Appsignal.set_gauge("erlang_schedulers", :erlang.system_info(:schedulers), %{kind: "Total"})
+
+    Appsignal.set_gauge(
+      "erlang_schedulers",
+      :erlang.system_info(:schedulers_online),
+      %{kind: "Online"}
+    )
+  end
+
+  defp process_metrics() do
+    Appsignal.set_gauge("erlang_processes", :erlang.system_info(:process_limit), %{kind: "Limit"})
+    Appsignal.set_gauge("erlang_processes", :erlang.system_info(:process_count), %{kind: "Count"})
```

Comment:
we probably want to lowercase tag values for consistency

Reply:
👍